src/server/utils/query/log-query.ts

```diff
@@ -65,10 +65,11 @@ function timeVariables(essence: Essence, timekeeper: Timekeeper): Record<string,
   return variables;
 }
 
-export function logQueryInfo(essence: Essence, timekeeper: Timekeeper, logger: Logger, executionTime: number) {
+export function logQueryInfo(essence: Essence, timekeeper: Timekeeper, logger: Logger, executionTime: number, context: Record<string, unknown>) {
   const nonTimeFilters = essence.filter.removeClause(essence.getTimeDimension().name);
 
   logger.log(`Visualization query ${essence.description(timekeeper)}`, {
+    ...context,
```

Comment:
Could we make some guards? E.g remove all undefined object properties.

Reply:
Actually, `undefined` is an invalid json value so it won't be logged due to `JSON.stringify` call. I'm not sure if I want to fiddle with this. It won't keep us safe in any way but will hinder debugging experience when some keys sometimes stop showing because they are `null`

Reply:
Good point, be transparent here.